apcd_cms/src/apps/registrations/templates/submission_form/registration_form_body.html

```diff
@@ -463,7 +463,7 @@ <h6>
                         class="integerfield"
                         id="total_covered_lives_{{ent_no}}_{{r.reg_id}}"
                         inputmode="numeric"
-                        pattern="^[1-9]+[0-9]*$"
+                        pattern="^[0-9]*$"
```

Comment:
can you use this instead : `^(0|[1-9][0-9]*)$`

`"^[0-9]*$"` allows a string like 0000. Instead we only want a string which is valid number.